### DISCUSSION TITLE
Transposed table (axises flipped)

### DISCUSSION THREAD

**Discussion by villesau** [chakra-ui/chakra-ui#5972] [Ideas]
### Description

Chakra-ui supports tables in this format:

```
+--------+--------+--------+
|  Col1  |  Col2  |  Col3  |
+--------+--------+--------+
| Val 1  | Val 2  | Val 3  |
| Val 12 | Val 22 | Val 33 |
+--------+--------+--------+
```

However, would be good if tables could be supported in this format too:

```
+------+---------------+
| Col1 | Val 1  Val 12 |
+------+---------------+
| Col2 | Val 2  Val 22 |
+------+---------------+
| Col3 | Val 3  Val 33 |
+------+---------------+

```

### Problem Statement/Justification

There are valid UIs when this is desirable format.

### Proposed Solution or API

`<Table transposed>...</Table>`

### Alternatives

-

### Additional Information

https://stackoverflow.com/questions/6297591/how-to-invert-transpose-the-rows-and-columns-of-an-html-table
https://siongui.github.io/2017/04/16/css-only-transpose-html-table/
https://dbushell.com/2016/03/04/css-only-responsive-tables/

**Comment by segunadebayo** [comment]
Hi @villesau,

This sounds like a pretty idea for a "recipe", not something we're looking to add to Chakra UI by default.

Feel free to create this and post your solution, we'll add it to the docs.
